CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) to get started.
 
+## 9.5.0-beta.1 - September 14, 2020
```

Comment:
The beta.1 changelog should be merged to 9.5.0-alpha.1

Reply:
Got it, fixed!